src/commands/schemas.ts

```diff
@@ -55,13 +57,40 @@ function uploadVersionCommand(item: any) {
   );
 }
 
+async function openLatestSchemasCommand(topic: KafkaTopic) {
+  const highestVersionedSchemas = await getLatestSchemasForTopic(topic);
```

Comment:
Any reason we wouldn't want to return an empty array and explicitly show info/error notifications in `getLatestSchemasForTopic()` instead of throwing errors here?

Reply:
Will meet you in the middle and have explicit exception class for 'did not get any schemas when we really expected them', and then the calling command catches and "handles" with an error notification.

This leaves the interior function UI-independent and more easily testable.

Reply:
K, done. Now raises Error when called with grossly unexpected condition indicating programmer errors (!topic.hasSchema, or is local schema (at this time)), which will fall through uncaught and log sentry. All other issues raise CannotLoadSchemasError and will caught and be displayed to user, but still for most part ought not happen.